coin-flip/script.js

```diff
@@ -2,12 +2,13 @@ function getRandNumber(max) {
 	return Math.floor(Math.random() * max);
 }
 
-var number = getRandNumber(2);
 
-var headsOrTails = "None";
+const flipCoin = () => {
+	const number = getRandNumber(2);
 
-if (number == 1) {
-	document.getElementById("coin").innerHTML = "<h1>You flipped Heads!</h1>";
-} else {
-	document.getElementById("coin").innerHTML = "<h1>You flipped Tails!</h1>";
+	const headsOrTails = number === 1 ? "Heads" : "Tails";
+
+	const newHtml = `<h1>You flipped ${headsOrTails}!`;
+
+	document.getElementById("coinimg").innerHTML = newHtml;
```

Comment:
After all that, the main issue with your code is that your `<div>` ID value was changed. The JS was looking for `coin` and the `<div>` had `coinimg`.